### DISCUSSION TITLE
Table in all editor

### DISCUSSION THREAD

**Discussion by VladimirAlchin** [requarks/wiki#4797] [Error / Bug Report]
![screen](https://user-images.githubusercontent.com/77410395/146552435-8b3574ce-760c-49b6-aea8-5d19ac5614ed.png)
Good afternoon, in the current release, markdown editor does not support tables. The button is not active. 
Is it possible to get a table with the ability to sort in some of the editors?

When is the third version expected to be released?

my ver wiki 2.5.256

**Comment by NGPixel** [comment]
Tables are supported using standard markdown syntax. See https://www.markdownguide.org/extended-syntax/#tables

**Reply by VladimirAlchin** [reply]
I need the ability to create a table with sorting functions. Is it possible? 
When do you plan to add these features?
Is it possible to add your own js modifications? addons?

**Reply by NGPixel** [reply]
No, markdown tables don't have sorting capabilities.

You can add custom JS from the admin > theme (global) or from the page properties panel (per page).

**Reply by VladimirAlchin** [reply]
I would really like to get a table with the ability to sort and the tool draw.io in one editor.
How soon will new editors add?
